packages/jest-emotion/types/enzyme.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="jest" />
```

Comment:
If you need help with adding types for this - let me know.

Reply:
Let me take a stab, and you can review.